assets/base.css

```diff
@@ -1534,26 +1590,33 @@ details[open] > .share-button__fallback {
   color: rgba(var(--color-foreground), 0.75);
 }
 
-.field__input:hover,
-.select__select:hover,
-.customer .field input:hover,
-.customer select:hover,
-.localization-form__select:hover {
-  box-shadow: 0 0 0 calc(var(--inputs-border-width) + 1px) rgba(var(--color-foreground), var(--inputs-border-opacity));
+.field:hover.field:after,
+.select:hover.select:after,
+.select__select:hover.select__select:after,
+.customer .field:hover.field:after,
+.customer select:hover.select:after,
+.localization-form__select:hover.localization-form__select:after {
+  box-shadow: 0 0 0 calc(0.3rem + var(--inputs-border-width)) rgba(var(--color-foreground),var(--inputs-border-opacity));
+  border-radius: var(--inputs-radius);
 }
 
 .field__input:focus,
 .select__select:focus,
 .customer .field input:focus,
 .customer select:focus,
 .localization-form__select:focus {
-  box-shadow: 0 0 0 calc(var(--inputs-border-width) + 2px) rgba(var(--color-foreground), var(--inputs-border-opacity));
-  outline: transparent;
+  outline: 0.2rem solid rgba(var(--color-foreground),.5);
+  outline-offset: calc(0.3em + var(--inputs-border-width));
+  box-shadow: 0 0 0 0.3rem rgb(var(--color-background)),0 0 0.5rem 0.4rem rgba(var(--color-foreground),.3);
+}
+
+.localization-form__select:focus {
+  outline-offset: 0.3em;
 }
 
 .text-area,
```

Comment:
On the contact form it seems like the "comment" part is a little too close to the border compared to the other fields.
![](https://screenshot.click/22-01-t5ign-qgh1h.png)